### PR TITLE
arcticdb v4.4.0

### PR DESCRIPTION
**Pull request by IvoDD**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/man-group/ArcticDB/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: 246e497a5ecae04d012172bb066082b26d69f228caddd492ec1fbb19db9802f8
+  sha256: 11a307d8d05c5d4c7cb57a3357d5f3eccc273498dde8a57beebc373db82141c7
   patches:
     - patches/0001-Include-missing-headers.patch
```
Comment (jjerphan) [inline recipe/meta.yaml:12]: This patch is not needed anymore. Removing it will fix the error on the CI.

**Review by jjerphan — COMMENTED** [review]
Just one adaptation is required.

**Comment by IvoDD** [comment]
@conda-forge-admin, please rerender

**Comment by IvoDD** [comment]
@conda-forge-admin, please rerender

**Review by jjerphan — APPROVED** [review]
LGTM given 🟢 CI checks.

**Review by alexowens90 — APPROVED** [review]

**Pull request merged**